# Combine & EDA : Cenus & MBS Datasets

This script combines non pivot version of MBS and Census datasets, making it easier to manage columns and EDA.

The MBS dataset contains service level 1,2 and 3

In [1]:
import pandas as pd
import numpy as np
import os
import seaborn as sns

## Import Census and MBS Datasets

### Import Census

In [2]:
# import the census and assign to a dataframe

# setup path to original dataset
path = r"/Users/patel/Documents/CF-Data Anaylst Course/portfolio_projects/mbs_analysis/datasets/"
df_census_2014_22 = pd.read_pickle(
    os.path.join(
        path, "clean_datasets/census_data/2014_22_census_complelete_standardized.pkl"
    )
)
df_census_2014_22.info(10)

<class 'pandas.core.frame.DataFrame'>
Int64Index: 3222 entries, 3 to 4295
Data columns (total 25 columns):
 #   Column                   Non-Null Count  Dtype  
---  ------                   --------------  -----  
 0   SA3                      3222 non-null   object 
 1   Year                     3222 non-null   int64  
 2   age_25-44                3222 non-null   float64
 3   age_45-64                3222 non-null   float64
 4   negative_income          3222 non-null   float64
 5   no_income                3222 non-null   float64
 6   average_income_$5200     3222 non-null   float64
 7   average_income_$13000    3222 non-null   float64
 8   average_income_$18200    3222 non-null   float64
 9   average_income_$26000    3222 non-null   float64
 10  average_income_$36400    3222 non-null   float64
 11  average_income_$46800    3222 non-null   float64
 12  average_income_$58500    3222 non-null   float64
 13  average_income_$71500    3222 non-null   float64
 14  average_income_$91000   

### Pivot Demograhic Columns to Rows

In [3]:
# defining columns to pivot
age_gender_p_columns = [
    "age_0-24",
    "age_25-44",
    "age_45-64",
    "age_65+",
    "male_pop",
    "female_pop",
    "total_population",
]
# defining columns to remain as is (id columns). expected these to duplicate
id_columns = [
    "key",
    "SA3",
    "Year",
    "negative_income",
    "no_income",
    "not_stated",
    "not_applicable",
    "average_income_$5200",
    "average_income_$13000",
    "average_income_$18200",
    "average_income_$26000",
    "average_income_$36400",
    "average_income_$46800",
    "average_income_$58500",
    "average_income_$71500",
    "average_income_$91000",
    "average_income_$130000",
    "average_income_$169000+",
]

# using melt function to pivot the demograhic into rows, naming columns Population_demograhic. Values stored in Population
df_census_2014_22_pivot = df_census_2014_22.melt(
    id_vars=id_columns,
    value_vars=age_gender_p_columns,
    var_name="Population_demographic",
    value_name="Population",
)
df_census_2014_22_pivot

key    SA3  Year  negative_income  no_income  not_stated  \
0      2014-10102  10102  2014            166.2     2839.0      3787.6   
1      2015-10102  10102  2015            163.6     2951.0      4047.8   
2      2016-10102  10102  2016            161.0     3063.0      4308.0   
3      2017-10102  10102  2017            180.4     3075.8      4085.0   
4      2018-10102  10102  2018            199.8     3088.6      3862.0   
...           ...    ...   ...              ...        ...         ...   
22549  2018-99999  99999  2018              3.6        5.4         4.8   
22550  2019-99999  99999  2019              2.4        3.6         4.2   
22551  2020-99999  99999  2020              1.2        1.8         3.6   
22552  2021-99999  99999  2021              0.0        0.0         3.0   
22553  2022-99999  99999  2022              0.0        0.0         3.0   

       not_applicable  average_income_$5200  average_income_$13000  \
0             11151.0                2013.8                 2376.8   
1             11193.0                1855.4                 2282.4   
2             11235.0                1697.0                 2188.0   
3             11516.6                1652.0                 2109.6   
4             11798.2                1607.0                 2031.2   
...               ...                   ...                    ...   
22549             5.4                   0.0                    2.4   
22550             3.6                   0.0                    1.6   
22551             1.8                   0.0                    0.8   
22552             0.0                   0.0                    0.0   
22553             0.0                   0.0                    0.0   

       average_income_$18200  average_income_$26000  average_income_$36400  \
0                     2815.2                 4824.6                 3607.8   
1                     2736.6                 5106.8                 3397.4   
2                     2658.0                 5389.0                 3187.0   
3                     2618.4                 5423.6                 3129.6   
4                     2578.8                 5458.2                 3072.2   
...                      ...                    ...                    ...   
22549                    0.0                    7.8                    3.0   
22550                    0.0                    5.2                    3.0   
22551                    0.0                    2.6                    3.0   
22552                    0.0                    0.0                    3.0   
22553                    0.0                    0.0                    3.0   

       average_income_$46800  average_income_$58500  average_income_$71500  \
0                     3896.2                 4438.8                 3594.6   
1                     3908.6                 4444.4                 3616.8   
2                     3921.0                 4450.0                 3639.0   
3                     3905.0                 4543.2                 3768.8   
4                     3889.0                 4636.4                 3898.6   
...                      ...                    ...                    ...   
22549                    0.0                    1.8                    0.0   
22550                    0.0                    1.2                    0.0   
22551                    0.0                    0.6                    0.0   
22552                    0.0                    0.0                    0.0   
22553                    0.0                    0.0                    0.0   

       average_income_$91000  average_income_$130000  average_income_$169000+  \
0                     5117.2                  4088.6                      0.0   
1                     5324.6                  4089.8                      0.0   
2                     5532.0                  4091.0                   1812.0   
3                     5993.8                  4637.2                   2182.4   
4    

In [4]:
# checking values before and after pivots
df_census_2014_22[df_census_2014_22["key"] == "2014-10102"]

SA3  Year  age_25-44  age_45-64  negative_income  no_income  \
3  10102  2014    15725.0    15715.2            166.2     2839.0   

   average_income_$5200  average_income_$13000  average_income_$18200  \
3                2013.8                 2376.8                 2815.2   

   average_income_$26000  ...  average_income_$130000  not_stated  \
3                 4824.6  ...                  4088.6      3787.6   

   not_applicable  male_pop  female_pop  total_population  \
3         11151.0   28038.8     27765.2           55804.2   

   average_income_$169000+  age_0-24  age_65+         key  
3                      0.0   17982.0   6378.4  2014-10102  

[1 rows x 25 columns]

In [5]:
df_census_2014_22_pivot[df_census_2014_22_pivot["key"] == "2014-10102"]

key    SA3  Year  negative_income  no_income  not_stated  \
0      2014-10102  10102  2014            166.2     2839.0      3787.6   
3222   2014-10102  10102  2014            166.2     2839.0      3787.6   
6444   2014-10102  10102  2014            166.2     2839.0      3787.6   
9666   2014-10102  10102  2014            166.2     2839.0      3787.6   
12888  2014-10102  10102  2014            166.2     2839.0      3787.6   
16110  2014-10102  10102  2014            166.2     2839.0      3787.6   
19332  2014-10102  10102  2014            166.2     2839.0      3787.6   

       not_applicable  average_income_$5200  average_income_$13000  \
0             11151.0                2013.8                 2376.8   
3222          11151.0                2013.8                 2376.8   
6444          11151.0                2013.8                 2376.8   
9666          11151.0                2013.8                 2376.8   
12888         11151.0                2013.8                 2376.8   
16110         11151.0                2013.8                 2376.8   
19332         11151.0                2013.8                 2376.8   

       average_income_$18200  average_income_$26000  average_income_$36400  \
0                     2815.2                 4824.6                 3607.8   
3222                  2815.2                 4824.6                 3607.8   
6444                  2815.2                 4824.6                 3607.8   
9666                  2815.2                 4824.6                 3607.8   
12888                 2815.2                 4824.6                 3607.8   
16110                 2815.2                 4824.6                 3607.8   
19332                 2815.2                 4824.6                 3607.8   

       average_income_$46800  average_income_$58500  average_income_$71500  \
0                     3896.2                 4438.8                 3594.6   
3222                  3896.2                 4438.8                 3594.6   
6444                  3896.2                 4438.8                 3594.6   
9666                  3896.2                 4438.8                 3594.6   
12888                 3896.2                 4438.8                 3594.6   
16110                 3896.2                 4438.8                 3594.6   
19332                 3896.2                 4438.8                 3594.6   

       average_income_$91000  average_income_$130000  average_income_$169000+  \
0                     5117.2                  4088.6                      0.0   
3222                  5117.2                  4088.6                      0.0   
6444                  5117.2                  4088.6                      0.0   
9666                  5117.2                  4088.6                      0.0   
12888                 5117.2                  4088.6                      0.0   
16110                 5117.2                  4088.6                      0.0   
19332                 5117.2                  4088.6                      0.0   

      Population_demographic  Population  
0                   age_0-24     17982.0  
3222               age_25-44     15725.0  
6444               age_45-64     15715.2  
9666                 age_65+      6378.4  
12888               male_pop     28038.8  
16110             female_pop     27765.2  
19332       total_population     55804.2

In [6]:
# extracting unique values, which need to be updated to reflect same as MBS for join
df_census_2014_22_pivot["Population_demographic"].unique()

array(['age_0-24', 'age_25-44', 'age_45-64', 'age_65+', 'male_pop',
       'female_pop', 'total_population'], dtype=object)

In [7]:
# Update demographic value to match the MBS dataset for left join
demographic_mapping = {
    "age_0-24": "0-24",
    "age_25-44": "25-44",
    "age_45-64": "45-64",
    "age_65+": "65+",
    "male_pop": "Males",
    "female_pop": "Females",
    "total_population": "All persons",
}
df_census_2014_22_pivot["Population_demographic"] = df_census_2014_22_pivot[
    "Population_demographic"
].map(demographic_mapping)
df_census_2014_22_pivot["Population_demographic"].unique()

array(['0-24', '25-44', '45-64', '65+', 'Males', 'Females', 'All persons'],
      dtype=object)

In [8]:
df_census_2014_22_pivot[df_census_2014_22_pivot["key"] == "2014-10102"]

key    SA3  Year  negative_income  no_income  not_stated  \
0      2014-10102  10102  2014            166.2     2839.0      3787.6   
3222   2014-10102  10102  2014            166.2     2839.0      3787.6   
6444   2014-10102  10102  2014            166.2     2839.0      3787.6   
9666   2014-10102  10102  2014            166.2     2839.0      3787.6   
12888  2014-10102  10102  2014            166.2     2839.0      3787.6   
16110  2014-10102  10102  2014            166.2     2839.0      3787.6   
19332  2014-10102  10102  2014            166.2     2839.0      3787.6   

       not_applicable  average_income_$5200  average_income_$13000  \
0             11151.0                2013.8                 2376.8   
3222          11151.0                2013.8                 2376.8   
6444          11151.0                2013.8                 2376.8   
9666          11151.0                2013.8                 2376.8   
12888         11151.0                2013.8                 2376.8   
16110         11151.0                2013.8                 2376.8   
19332         11151.0                2013.8                 2376.8   

       average_income_$18200  average_income_$26000  average_income_$36400  \
0                     2815.2                 4824.6                 3607.8   
3222                  2815.2                 4824.6                 3607.8   
6444                  2815.2                 4824.6                 3607.8   
9666                  2815.2                 4824.6                 3607.8   
12888                 2815.2                 4824.6                 3607.8   
16110                 2815.2                 4824.6                 3607.8   
19332                 2815.2                 4824.6                 3607.8   

       average_income_$46800  average_income_$58500  average_income_$71500  \
0                     3896.2                 4438.8                 3594.6   
3222                  3896.2                 4438.8                 3594.6   
6444                  3896.2                 4438.8                 3594.6   
9666                  3896.2                 4438.8                 3594.6   
12888                 3896.2                 4438.8                 3594.6   
16110                 3896.2                 4438.8                 3594.6   
19332                 3896.2                 4438.8                 3594.6   

       average_income_$91000  average_income_$130000  average_income_$169000+  \
0                     5117.2                  4088.6                      0.0   
3222                  5117.2                  4088.6                      0.0   
6444                  5117.2                  4088.6                      0.0   
9666                  5117.2                  4088.6                      0.0   
12888                 5117.2                  4088.6                      0.0   
16110                 5117.2                  4088.6                      0.0   
19332                 5117.2                  4088.6                      0.0   

      Population_demographic  Population  
0                       0-24     17982.0  
3222                   25-44     15725.0  
6444                   45-64     15715.2  
9666                     65+      6378.4  
12888                  Males     28038.8  
16110                Females     27765.2  
19332            All persons     55804.2

In [9]:
# import the mbs file and assign to a dataframe

df_mbs_2014_23 = pd.read_pickle(
    os.path.join(
        path, "clean_datasets/mbs_data/2014-22_mbs_state_complete_no_pivot.pkl"
    )
)
df_mbs_2014_23.info()

<class 'pandas.core.frame.DataFrame'>
Int64Index: 229569 entries, 0 to 258533
Data columns (total 22 columns):
 #   Column                         Non-Null Count   Dtype  
---  ------                         --------------   -----  
 0   Year                           229569 non-null  int64  
 1   StateTerritory                 229569 non-null  object 
 2   GeographicCode                 229569 non-null  object 
 3   GeographicAreaName             229569 non-null  object 
 4   GeographicGroup                229569 non-null  object 
 5   ServiceLevel                   229569 non-null  object 
 6   Service                        229569 non-null  object 
 7   DemographicGroup               229569 non-null  object 
 8   MBS_per_100                    229569 non-null  float64
 9   No_of_patients                 229569 non-null  Int32  
 10  No_of_services                 229569 non-null  Int64  
 11  %_People_had_service           229569 non-null  float64
 12  Services_100_people           

## Combine Census & MBS

Census will be combined with MBS using left join. Key columns will be Year, GeographicCode (SA3), DemograhicGroup (Population Demographic)

In [10]:
df_census_2014_22_pivot.shape

(22554, 20)

In [11]:
df_mbs_2014_23.shape

(229569, 22)

Expected 229,569 rows and 42 columns - number of rows will be the same as MBS. No nulls expected

In [12]:
# left join Cenus to MBS
df_mbs_census_combined_np = df_mbs_2014_23.merge(
    df_census_2014_22_pivot,
    how="left",
    left_on=["Year", "GeographicCode", "DemographicGroup"],
    right_on=["Year", "SA3", "Population_demographic"],
    indicator=True,
)
df_mbs_census_combined_np.shape

(229569, 42)

In [13]:
# checking for any left only joins
df_mbs_census_combined_np["_merge"].value_counts(dropna=False)

both          229569
left_only          0
right_only         0
Name: _merge, dtype: int64

In [14]:
# checking for nulls
df_mbs_census_combined_np.isna().sum()

Year                             0
StateTerritory                   0
GeographicCode                   0
GeographicAreaName               0
GeographicGroup                  0
ServiceLevel                     0
Service                          0
DemographicGroup                 0
MBS_per_100                      0
No_of_patients                   0
No_of_services                   0
%_People_had_service             0
Services_100_people              0
Total_mbs_paid_$                 0
Total_provider_fees_$            0
ERP                              0
key_x                            0
Out_of_Pocket                    0
Out_of_pocket_cost_%             0
Out_of_pocket_cost_per_person    0
No_of_service_per_person         0
Patient_ERP_Flag                 0
key_y                            0
SA3                              0
negative_income                  0
no_income                        0
not_stated                       0
not_applicable                   0
average_income_$5200

In [15]:
# checking specific key to ensure the join has worked
df_mbs_census_combined_np[
    (df_mbs_census_combined_np["key_x"] == "2014-10102")
    & (df_mbs_census_combined_np["Service"] == "Allied Health attendances (total)")
]

Year StateTerritory GeographicCode GeographicAreaName  \
508  2014            NSW          10102         Queanbeyan   
509  2014            NSW          10102         Queanbeyan   
510  2014            NSW          10102         Queanbeyan   
511  2014            NSW          10102         Queanbeyan   
512  2014            NSW          10102         Queanbeyan   
513  2014            NSW          10102         Queanbeyan   
514  2014            NSW          10102         Queanbeyan   

               GeographicGroup ServiceLevel  \
508  Major cities - higher SES      Level 1   
509  Major cities - higher SES      Level 1   
510  Major cities - higher SES      Level 1   
511  Major cities - higher SES      Level 1   
512  Major cities - higher SES      Level 1   
513  Major cities - higher SES      Level 1   
514  Major cities - higher SES      Level 1   

                               Service DemographicGroup  MBS_per_100  \
508  Allied Health attendances (total)             0-24       2534.0   
509  Allied Health attendances (total)            25-44       3160.0   
510  Allied Health attendances (total)            45-64       3924.0   
511  Allied Health attendances (total)              65+       5215.0   
512  Allied Health attendances (total)      All persons       3390.0   
513  Allied Health attendances (total)          Females       4240.0   
514  Allied Health attendances (total)            Males       2554.0   

     No_of_patients  ...  average_income_$36400  average_income_$46800  \
508            3536  ...                 3607.8                 3896.2   
509            3377  ...                 3607.8                 3896.2   
510            5547  ...                 3607.8                 3896.2   
511            3128  ...                 3607.8                 3896.2   
512           15587  ...                 3607.8                 3896.2   
513            9172  ...                 3607.8                 3896.2   
514            6415  ...                 3607.8                 3896.2   

     average_income_$58500  average_income_$71500  average_income_$91000  \
508                 4438.8                 3594.6                 5117.2   
509                 4438.8                 3594.6                 5117.2   
510                 4438.8                 3594.6                 5117.2   
511                 4438.8                 3594.6                 5117.2   
512                 4438.8                 3594.6                 5117.2   
513                 4438.8                 3594.6                 5117.2   
514                 4438.8                 3594.6                 5117.2   

     average_income_$130000 average_income_$169000+  Population_demographic  \
508                  4088.6                     0.0                    0-24   
509                  4088.6                     0.0                   25-44   
510                  4088.6                     0.0                   45-64   
511                  4088.6                     0.0                     65+   
512                  4088.6                     0.0             All persons   
513                  4088.6                     0.0                 Females   
514                  4088.6                     0.0                   Males   

     Population  _merge  
508     17982.0    both  
509     15725.0    both  
510     15715.2    both  
511      6378.4    both  
512     55804.2    both  
513     27765.2    both  
514     28038.8    both  

[7 rows x 42 columns]

### Drop Unrequired Columns

In [16]:
# checking column that have duplicated or not required to drop
df_mbs_census_combined_np.columns

Index(['Year', 'StateTerritory', 'GeographicCode', 'GeographicAreaName',
       'GeographicGroup', 'ServiceLevel', 'Service', 'DemographicGroup',
       'MBS_per_100', 'No_of_patients', 'No_of_services',
       '%_People_had_service', 'Services_100_people', 'Total_mbs_paid_$',
       'Total_provider_fees_$', 'ERP', 'key_x', 'Out_of_Pocket',
       'Out_of_pocket_cost_%', 'Out_of_pocket_cost_per_person',
       'No_of_service_per_person', 'Patient_ERP_Flag', 'key_y', 'SA3',
       'negative_income', 'no_income', 'not_stated', 'not_applicable',
       'average_income_$5200', 'average_income_$13000',
       'average_income_$18200', 'average_income_$26000',
       'average_income_$36400', 'average_income_$46800',
       'average_income_$58500', 'average_income_$71500',
       'average_income_$91000', 'average_income_$130000',
       'average_income_$169000+', 'Population_demographic', 'Population',
       '_merge'],
      dtype='object')

In [17]:
# dropping merge, key_y and SA3 columns
df_mbs_census_combined_np.drop(columns=["_merge", "key_y", "SA3"], inplace=True)
df_mbs_census_combined_np.columns

Index(['Year', 'StateTerritory', 'GeographicCode', 'GeographicAreaName',
       'GeographicGroup', 'ServiceLevel', 'Service', 'DemographicGroup',
       'MBS_per_100', 'No_of_patients', 'No_of_services',
       '%_People_had_service', 'Services_100_people', 'Total_mbs_paid_$',
       'Total_provider_fees_$', 'ERP', 'key_x', 'Out_of_Pocket',
       'Out_of_pocket_cost_%', 'Out_of_pocket_cost_per_person',
       'No_of_service_per_person', 'Patient_ERP_Flag', 'negative_income',
       'no_income', 'not_stated', 'not_applicable', 'average_income_$5200',
       'average_income_$13000', 'average_income_$18200',
       'average_income_$26000', 'average_income_$36400',
       'average_income_$46800', 'average_income_$58500',
       'average_income_$71500', 'average_income_$91000',
       'average_income_$130000', 'average_income_$169000+',
       'Population_demographic', 'Population'],
      dtype='object')

## Derive %OOP from Income

In [18]:
df_mbs_census_combined_np_new = df_mbs_census_combined_np.copy()

In [19]:
df_mbs_census_combined_np_new["%_out_of_pocket_by_$5200"] = (
    df_mbs_census_combined_np_new["Out_of_pocket_cost_per_person"] / 5200
) * 100
df_mbs_census_combined_np_new["%_out_of_pocket_by_$13000"] = (
    df_mbs_census_combined_np_new["Out_of_pocket_cost_per_person"] / 13000
) * 100
df_mbs_census_combined_np_new["%_out_of_pocket_by_$18200"] = (
    df_mbs_census_combined_np_new["Out_of_pocket_cost_per_person"] / 18200
) * 100
df_mbs_census_combined_np_new["%_out_of_pocket_by_$26000"] = (
    df_mbs_census_combined_np_new["Out_of_pocket_cost_per_person"] / 26000
) * 100
df_mbs_census_combined_np_new["%_out_of_pocket_by_$36400"] = (
    df_mbs_census_combined_np_new["Out_of_pocket_cost_per_person"] / 36400
) * 100
df_mbs_census_combined_np_new["%_out_of_pocket_by_$46800"] = (
    df_mbs_census_combined_np_new["Out_of_pocket_cost_per_person"] / 46800
) * 100
df_mbs_census_combined_np_new["%_out_of_pocket_by_$58500"] = (
    df_mbs_census_combined_np_new["Out_of_pocket_cost_per_person"] / 58500
) * 100
df_mbs_census_combined_np_new["%_out_of_pocket_by_$71500"] = (
    df_mbs_census_combined_np_new["Out_of_pocket_cost_per_person"] / 71500
) * 100
df_mbs_census_combined_np_new["%_out_of_pocket_by_$91000"] = (
    df_mbs_census_combined_np_new["Out_of_pocket_cost_per_person"] / 91000
) * 100
df_mbs_census_combined_np_new["%_out_of_pocket_by_$130000"] = (
    df_mbs_census_combined_np_new["Out_of_pocket_cost_per_person"] / 130000
) * 100
df_mbs_census_combined_np_new["%_out_of_pocket_by_$169000+"] = (
    df_mbs_census_combined_np_new["Out_of_pocket_cost_per_person"] / 169000
) * 100

In [20]:
df_mbs_census_combined_np_new.head(3)

Year StateTerritory GeographicCode GeographicAreaName  \
0  2014            ACT          80101          Belconnen   
1  2014            ACT          80101          Belconnen   
2  2014            ACT          80101          Belconnen   

             GeographicGroup ServiceLevel                            Service  \
0  Major cities - medium SES      Level 1  Allied Health attendances (total)   
1  Major cities - medium SES      Level 1  Allied Health attendances (total)   
2  Major cities - medium SES      Level 1  Allied Health attendances (total)   

  DemographicGroup  MBS_per_100  No_of_patients  ...  \
0             0-24       2576.0            5624  ...   
1            25-44       4004.0            7714  ...   
2            45-64       4672.0            8998  ...   

   %_out_of_pocket_by_$13000  %_out_of_pocket_by_$18200  \
0                   0.257037                   0.183598   
1                   0.352197                   0.251569   
2                   0.153768                   0.109835   

   %_out_of_pocket_by_$26000  %_out_of_pocket_by_$36400  \
0                   0.128519                   0.091799   
1                   0.176098                   0.125785   
2                   0.076884                   0.054917   

   %_out_of_pocket_by_$46800  %_out_of_pocket_by_$58500  \
0                   0.071399                   0.057119   
1                   0.097832                   0.078266   
2                   0.042713                   0.034171   

  %_out_of_pocket_by_$71500  %_out_of_pocket_by_$91000  \
0                  0.046734                   0.036720   
1                  0.064036                   0.050314   
2                  0.027958                   0.021967   

   %_out_of_pocket_by_$130000  %_out_of_pocket_by_$169000+  
0                    0.025704                     0.019772  
1                    0.035220                     0.027092  
2                    0.015377                     0.011828  

[3 rows x 50 columns]

In [21]:
df_mbs_census_combined_np_new.isna().sum()

Year                             0
StateTerritory                   0
GeographicCode                   0
GeographicAreaName               0
GeographicGroup                  0
ServiceLevel                     0
Service                          0
DemographicGroup                 0
MBS_per_100                      0
No_of_patients                   0
No_of_services                   0
%_People_had_service             0
Services_100_people              0
Total_mbs_paid_$                 0
Total_provider_fees_$            0
ERP                              0
key_x                            0
Out_of_Pocket                    0
Out_of_pocket_cost_%             0
Out_of_pocket_cost_per_person    0
No_of_service_per_person         0
Patient_ERP_Flag                 0
negative_income                  0
no_income                        0
not_stated                       0
not_applicable                   0
average_income_$5200             0
average_income_$13000            0
average_income_$1820

### Export MBS Census Combined Dataset

In [22]:
df_mbs_census_combined_np_new.to_pickle(
    os.path.join(path, "clean_datasets/2014_22_mbs_cenus_combined_no_pivot.pkl")
)